src/commands.jl

```diff
@@ -86,13 +86,13 @@ finish_stack!(frame::Frame, istoplevel::Bool=false) = finish_stack!(finish_and_r
     pc = next_until!(predicate, frame, istoplevel=false)
 
 Execute the current statement. Then step through statements of `frame` until the next
-statement satifies `predicate(stmt)`. `pc` will be the index of the statement at which
+statement satifies `predicate(frame)`. `pc` will be the index of the statement at which
```

Comment:
```suggestion
statement satisfies `predicate(frame)`. `pc` will be the index of the statement at which
```

(unrelated to your change, but happened to notice)